src/test/java/core/basesyntax/service/CsvWriterServiceImplTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CsvWriterServiceImplTest {
+    private static WriterService writerService;
+    private static final String PATH_FOR_TEST
+            = "src/test/java/resources/csvFileWriterTestResult.csv";
+    private static final String RESULT_PATH
+            = "src/test/java/resources/report.csv";
+    private static final String WRONG_RESULT_PATH
+            = "////";
+    private static final String INPUT = "fruit,quantity" + System.lineSeparator()
+            + "banana,152" + System.lineSeparator()
+            + "apple,90" + System.lineSeparator();
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        writerService = new CsvWriterServiceImpl();
+    }
+
+    @Test
+    public void writeToFileTestWithCorrectInput_Ok() {
```

Comment:
You can use Files.readAllLines() here as well

Reply:
done